static/frontend-tls-selfsigned/conf.d/default.conf

```diff
@@ -3,6 +3,7 @@ server {
     server_name _;
 
     root /var/www/public;
+    access_log /proc/self/fd/1;
```

Comment:
Is there a specific reason why you configure the `access_log` in each server instead of just in `http` of `nginx.conf` to set it globally?

Reply:
There is, yes - as you say, I agree that it shouldn't be different behaviour-wise from the global configuration, however some of the [`nginx` HTTP log documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log) mentions that it can be a good idea:

"It is thus a good idea to specify both [root](https://nginx.org/en/docs/http/ngx_http_core_module.html#root) and access_log on the same configuration level"

Reply:
I feel like the documentation is a bit vague on why it should be specified at the same level.
But it's nothing important anyways and the faster this gets merged the happier I am.

This config is certainly better than the hacks I am currently using.

Reply:
Changes merged; if you have a bit of time to test the changes out before an updated release, I'd appreciate it!